### PR TITLE
Lost connection after project reload

### PR DESCRIPTION
**Pull request by maltaesousa**
Reloads plugin's GUI after a project load.
Delete some backward compatibility stuff (a frozen version for 2.X is available)

### CODE REVIEW

File: selvansgeo.py [hunk 1]
```diff
@@ -293,9 +265,8 @@ def connectionsInit(self):
                 if checkdb.open():
                     self.credentialInstance.put(connectionInfo, user, pwd)
                 else:
-                    self.messageBar.pushMessage("Erreur",
-                                                str(u"Mauvais mot de passe"),
-                                                level=QgsMessageBar.CRITICAL)
+                    self.messageBar.pushCritical(
+                        "Erreur", str(u"Mauvais mot de passe"))
```
Comment (kalbermattenm) [inline selvansgeo.py]: In Python 3, if your file is defined as an UTF-8 encoded file (which it is, see file first line), you do not need to declare stuff as unicode anymore, thus IHMO, this should work:

```suggestion
                        "Erreur", "Mauvais mot de passe")
```

Comment (kalbermattenm) [reply selvansgeo.py]: And this could be applied everywhere... (but should first be tested...)

**Review by kalbermattenm — COMMENTED** [review]
Just one remark concerning unicode, utf-8 and gunicorn (look for the mistake...)

**Comment by maltaesousa** [comment]
Deployed and tested by Romain. Merging.

**Pull request merged**